packages/camera/jest.config.js

```diff
@@ -0,0 +1,199 @@
+/*
```

Comment:
you can remove all the useless comments from this file

Reply:
hi samy.in future some developer might need some configuration which he can uncomment and directly use it.thats why i have kept this comments and code

Reply:
If a developer needs some configuration, he can check the official documentation which is pretty extensive :) Please remove all the comments from this file